README.md

```diff
@@ -91,6 +147,36 @@ If you prefer a non-Christmas option, you may choose a different song with simil
 
 ---
 
+
+Solution:
+
+??? - though, I'm not sure if this is right? It says in the homework not to just 
+console.log each line.... and I don't think I did that per se, but I defininitly
+ am console.log'ing the solution, ultimately.)
+
+```
+var days = ["first ", "second ", "third ", "fourth ", "fifth ", "sixth ",
+ "seventh ", "eighth ", "ninth ", "tenth ", "eleventh ", "twelfth "]
+
+var gifts = ["A Partridge in a Pear Tree", "Two Turtles Doves", 
+"Three French Hens", "Four Calling Birds", "Five Golden Rings", 
+"Six Geese a Laying", "Seven Swans a Swimming", "Eight Maids a Milking", 
+"Nine Ladies Dancing", "Ten Lords a Leaping", "Eleven Pipers Piping", 
+"Twelve Drummers Drumming"]
+
+  function earlierDays(b){
+    for (g = b-2; g >= 0; g--) {
+      console.log(gifts[g] + ',' + '\n')}
+  };
+
+function singIt(){
+  for ( var i = 0; i <= days.length - 2; i++) {
+    console.log("On the " + days[i] + "day of Christmas" + '\n'
+     + "my true love sent to me: " + gifts[i] + '\n' + earlierDays(i));
+  };
+}
```

Comment:
Make sure to double-check the outputs. This one, for example, only runs to the start of the 11th day, doesn't finish it, and intersperses "undefined" throughout. The first problem is because you have a -2 instead of a -1 in your loop for `singIt()`, and the second problem is because you are `console.log()`ing the output of a function. When the `previousDays` function runs, it has the side effect of logging the lyrics to the song, but also returns the value `undefined`, which `singIt()` then dutifully logs.